### PR TITLE
Update to Liveview 1.0.0

### PR DESCRIPTION
**Pull request by DamascenoV**
This commit updates the phoenix_live_view dependency to the latest version (1.0.0).

Also update the Elixir version to the version 1.16 to remove the warning of the :tw_merge
image

Fix of broken tests.

## Summary by Sourcery

Update the phoenix_live_view dependency to version 1.0.0 and Elixir to version 1.16. Fix broken tests by refining assertions for improved clarity.

Enhancements:
- Update Elixir version to 1.16 to remove warnings related to :tw_merge.

Tests:
- Fix broken tests by updating assertions to use String.contains? for better readability and maintainability.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -5,7 +5,7 @@ defmodule SaladUI.MixProject do
     [
       app: :salad_ui,
       version: "0.14.1",
-      elixir: "~> 1.14",
```
Comment (jaeyson) [inline mix.exs:8]: Would it be better to leave the version as it is (1.14) and make `:tw_merge` downgrade Elixir version instead?

Comment (jaeyson) [reply mix.exs:8]: also I made [PR](https://github.com/bluzky/tw_merge/pull/1) in `:tw_merge` in sync with Phx v1.7.17's Elixir version (`1.14`)

Comment (DamascenoV) [reply mix.exs]: Ok, I'll keep this PR focused on the live_view update. Thanks!

**Review by bluzky — APPROVED** [review]

**Comment by bluzky** [comment]
Thanks @DamascenoV, @jaeyson  release 0.14.3 with Liveview 1.0 and tw_merge 1.0.1

**Pull request merged**